docs/draft/6-payloads.md

```diff
@@ -0,0 +1,359 @@
+| 1 | clock | `uint64` | The clock of the chat with the user |
+| 2 | ens_name | `string` | The ENS name if set |
+| 3 | profile_image | `string` | The base64 encoded profile picture of the user |
+
+#### Contact update
+
+A client SHOULD send a `ContactUpdate` to all the contacts each time:
+
+- The ens_name has changed
+- The profile image is edited
+
+A client SHOULD also periodically send a `ContactUpdate` to all the contacts, the interval is up to the client, the Status official client sends these updates every 48 hours.
+
+### SyncInstallationContact
+
+`SyncInstallationContact` messages are used to synchronize in a best-effort the contacts to other devices.
+
+```protobuf
+message SyncInstallationContact {
+  uint64 clock = 1;
+  string id = 2;
+  string profile_image = 3;
+  string ens_name = 4;
+  uint64 last_updated = 5;
+  repeated string system_tags = 6;
+}
+```
+
+
+#### Payload
+
+| Field | Name | Type | Description |
+| ----- | ---- | ---- | ---- |
+| 1 | clock | `uint64` | clock value of the chat | 
+| 2 | id | `string` | id of the contact synced |
+| 3 | profile_image | `string` |  `base64` encoded profile picture of the user |
+| 4 | ens_name | `string` | ENS name of the contact |
+| 5 | `array[string]` | Array of `system_tags` for the user, this can currently be: `":contact/added", ":contact/blocked", ":contact/request-received"`|
+
+### SyncInstallationPublicChat
+
+`SyncInstallationPublicChat` message is used to synchronize in a best-effort the public chats to other devices.
+
+```protobuf
+message SyncInstallationPublicChat {
+  uint64 clock = 1;
+  string id = 2;
+}
+```
+
+#### Payload
+
+| Field | Name | Type | Description |
+| ----- | ---- | ---- | ---- |
+| 1 | clock | `uint64` | clock value of the chat | 
+| 2 | id | `string` | id of the chat synced |
+
+### PairInstallation
+
+`PairInstallation` messages are used to propagate informations about a device to its paired devices.
+
+```protobuf
+message PairInstallation {
+  uint64 clock = 1;
+  string installation_id = 2;
+  string device_type = 3;
+  string name = 4;
+}
+```
+
+#### Payload
+
+| Field | Name | Type | Description |
+| ----- | ---- | ---- | ---- |
+| 1 | clock | `uint64` | clock value of the chat | 
+| 2| installation_id | `string` | A randomly generated id that identifies this device |
+| 3 | device_type | `string` | The OS of the device `ios`,`android` or `desktop` |
+| 4 | name | `string` | The self-assigned name of the device |
+
+### MembershipUpdateMessage and MembershipUpdateEvent
+
+`MembershipUpdateEvent` is a message used to propagate information about group membership changes in a group chat.
+The details are in the  [Group chats specs](status-group-chats-spec.md)
+
+## Upgradability
+
+There are two ways to upgrade the protocol without breaking compatibility:
+
+- Accretion is always supported
+- Deletion of existing fields or messages is not supported and might break compatibility
+
+## Security Considerations
+
+-
+
+## Design rationale
```

Comment:
## Copyright

Copyright and related rights waived via [CC0](https://creativecommons.org/publicdomain/zero/1.0/).

(realized this was missing in original, an oversight - possibly separate pr).